python3.hcl

```
@@ -9,6 +9,12 @@ env = {
   PATH: "${HERMIT_ENV}/.hermit/python/bin:${PATH}",
 }
 
+on unpack {
+  copy { from = "python3/relocate.sh" to = "${root}/relocate.sh" }
+  chmod { file = "${root}/relocate.sh" mode = 0755 }
+  run { cmd = "${root}/relocate.sh ${root}" }
```

```suggestion
  run { cmd = "python3/relocate.sh ${root}" }
```

Commit `relocate.sh` with `+x` and this can be a single run command?